pom.xml

```diff
@@ -40,7 +40,7 @@
         <swagger-annotations-version>1.6.0</swagger-annotations-version>
         <openapi-annotations-version>2.1.7</openapi-annotations-version>
         <!-- Used for liquibase maven plugin. Somehow use version from bom-internal instead -->
-        <postgresql.version>42.2.21</postgresql.version>
+        <postgresql.version>42.2.19</postgresql.version>
```

Comment:
Did you roll this back too on purpose (or maybe you just did a git revert)? Seems like we could still have the latest client, unless it's known not to work.

Reply:
this is just a git revert